inst/templates/cran/Makevars

```diff
@@ -24,9 +24,9 @@ $(STATLIB):
 	if [ "$(NOT_CRAN)" != "true" ]; then \
 		export CARGO_HOME=$(CARGOTMP); \
 	fi && \
-		export PATH="$(PATH):$(HOME)/.cargo/bin" && \
-			cargo build $(CRAN_FLAGS) --lib --release --manifest-path=./rust/Cargo.toml --target-dir $(TARGET_DIR) && \
-			echo `cargo --version` && echo `rustc --version`;
+        echo `cargo --version` && echo `rustc --version` && \
+            export PATH="$(PATH):$(HOME)/.cargo/bin" && \
```

Comment:
Should `PATH` still be updated first before `cargo` is invoked? It seems it will fail if `cargo` is not on the `PATH` by default